.github/workflows/verify.yaml

```diff
@@ -82,22 +82,14 @@ jobs:
     env:
       JACOCO: true
 
-    services:
-      postgres:
-        image: postgres:14.2
-        ports:
-          - 5432:5432
-        env:
-          POSTGRES_PASSWORD: password
-
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/gradle-setup
 
       - name: Postgresql Tests
         uses: ./.github/actions/run-tests
         with:
-          command: ./gradlew test -DincludeTags="PostgresqlIntegrationTest"
+          command: ./gradlew test -DincludeTags="ComponentTest"
```

Comment:
we should rename the job from `Postgresql-Integration-Tests` to `Component-Tests`